src/main/java/Duke.java

```diff
@@ -1,10 +1,86 @@
+        System.out.println("Hello! I'm Duke" + System.lineSeparator() + "What can I do for you?" + dividingLine);
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while(!line.equals("bye")){
+            if(line.equals("list")){
+                printList();
+            }
+            else if(line.contains("mark") && !line.contains("unmark")){
+                int taskNum = Integer.parseInt(line.substring(5)) - 1;
+                tasks[taskNum].markAsDone();
+                System.out.println(dividingLine + "Nice! I've marked this task as done:");
+                System.out.println("[" + tasks[taskNum].getStatusIcon() + "] " + tasks[taskNum].description
+                        + dividingLine);
+            }
+            else if(line.contains("unmark")){
+                int unmarkNum = Integer.parseInt(line.substring(7)) - 1;
+                tasks[unmarkNum].unmarkAsDone();
+                System.out.println(dividingLine + "OK, I've marked this task as not done yet:");
+                System.out.println("[" + tasks[unmarkNum].getStatusIcon() + "] " + tasks[unmarkNum].description
+                        + dividingLine);
+            }
+            else if(line.contains("todo")){
+                line = line.replace("todo ", "");
+                addTask(new Todo(line));
+                System.out.println(dividingLine + "Got it. I've added this task:");
+                System.out.println(tasks[taskCount].toString());
+                System.out.println("Now you have " + (taskCount + 1) + " tasks in the list." + dividingLine);
+                taskCount++;
+            }
+            else if(line.contains("deadline")){
+                line = line.replace("deadline ", "");
+                addTask(new Deadline(line.substring(0, line.indexOf(" /") - 1),
+                        line.substring(line.indexOf("/by ") + 4)));
+                System.out.println(dividingLine + "Got it. I've added this task:");
+                System.out.println(tasks[taskCount].toString());
+                System.out.println("Now you have " + (taskCount + 1) + " tasks in the list." + dividingLine);
+                taskCount++;
+            }
+            else if(line.contains("event")){
+                line = line.replace("event ", "");
+                addTask(new Event(line.substring(0, line.indexOf(" /") - 1),
+                        line.substring(line.indexOf("/from ") + 6, line.indexOf(" /to",
+                                line.indexOf("/from") - 1)),
+                        line.substring(line.indexOf("/to ") + 4)));
+                System.out.println(dividingLine + "Got it. I've added this task:");
+                System.out.println(tasks[taskCount].toString());
+                System.out.println("Now you have " + (taskCount + 1) +" tasks in the list." + dividingLine);
+                taskCount++;
+            }
+            else{
```

Comment:
This condition should throw an exception instead of showing that the task has been added